frontend/coreproject/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,156 @@
+<script lang="ts">
+	export let anime: any;
+
+	import ScrollArea from '$components/shared/ScrollArea.svelte';
+	import SidebarDetails from '$components/pages/anime_info/SidebarDetails.svelte';
+
+	// icons
+	import PlayCircle from '$icons/PlayCircle.svelte';
+	import Read from '$icons/Read.svelte';
+	import Listen from '$icons/Listen.svelte';
+	import Video from '$icons/Video.svelte';
+	import Download from '$icons/Download.svelte';
+	import Share from '$icons/Share.svelte';
+	import SettingsOutline from '$icons/SettingsOutline.svelte';
+	import Edit from '$icons/Edit.svelte';
+</script>
+
+<div class="grid grid-cols-12 items-start">
+	<div class="col-span-10 grid grid-cols-12 items-end">
+		<div class="col-span-7 flex items-end gap-10">
+			<img class="md:w-1/3 2xl:w-2/5 md:rounded-xl 2xl:rounded-3xl" src={anime.cardBackgroundImage} alt={anime.titles.eng} />
+			<div>
+				<h2 class="font-bold 2xl:!text-6xl">{anime.titles.eng}</h2>
+
+				<div class="anime_titles mt-3 flex items-center gap-2 md:text-xs 2xl:!text-lg font-medium opacity-80">
+					<span>{anime.titles.japanese}</span>
+					<span>-</span>
+					<span>{anime.titles.eng}</span>
+					<span>-</span>
+					<span>{anime.titles.others}</span>
+				</div>
+
+				<div
+					class="anime_basic_details mt-2 flex flex-wrap items-center gap-2 pr-5 md:text-xs 2xl:!text-lg 2xl:!leading-none font-semibold opacity-100"
+				>
+					<span>{anime.type}</span>
+					<span>-</span>
+					<span>{anime.episodes} eps</span>
+					<span>-</span>
+					<span>{anime.status}</span>
+					<span>-</span>
+					<span>{anime.premiered}</span>
+					<span>-</span>
+					<span>{anime.studio}</span>
+				</div>
+
+				<div class="md:mt-5 2xl:mt-10 flex items-center gap-5">
+					<button
+						type="button"
+						class="btn 2xl:btn-lg md:h-16 2xl:h-24 justify-start md:rounded-lg 2xl:rounded-xl bg-primary-500 md:text-xs 2xl:text-xl font-bold text-white"
+					>
+						<PlayCircle width="30" height="30" color="white" />
+						<div class="flex flex-col items-start">
+							<span class="leading-5">Watch</span>
+							<span class="md:text-[0.6rem] 2xl:text-lg font-normal leading-none opacity-70">Ep 01</span>
+						</div>
+					</button>
+
+					<button
+						type="button"
+						class="btn md:h-16 2xl:h-24 2xl:w-24 flex-col gap-1 md:rounded-lg 2xl:rounded-xl bg-secondary-100 md:text-xs 2xl:text-xl md:font-bold 2xl:font-semibold text-surface-500"
+					>
+						<Read width="22" height="22" color="bg-surface-500" />
+						Read
+					</button>
+
+					<button
+						type="button"
+						class="btn md:h-16 2xl:h-24 2xl:w-24 flex-col gap-1 md:rounded-lg 2xl:rounded-xl bg-secondary-100 md:text-xs 2xl:text-xl md:font-bold 2xl:font-semibold text-surface-500"
+					>
+						<Listen width="22" height="22" color="bg-surface-500" />
+						Listen
+					</button>
```

Comment:
These should be `h-20` on my screen

![image](https://user-images.githubusercontent.com/61817579/230699905-b650fa34-b219-4b28-94f3-7fb2a903c5de.png)